build.sbt

```diff
@@ -48,12 +49,12 @@ val playTestdata = Project("play-grpc-testdata", file("play-testdata"))
   .settings(
     scalacOptions += "-Xlint:-unused,_",  // can't do anything about unused things in generated code
     javacOptions -= "-Xlint:deprecation", // can't do anything about deprecations in generated code
-    akkaGrpcExtraGenerators ++= List(
-      akka.grpc.gen.javadsl.play.PlayJavaClientCodeGenerator,
-      akka.grpc.gen.javadsl.play.PlayJavaServerCodeGenerator,
-      akka.grpc.gen.scaladsl.ScalaMarshallersCodeGenerator,
-      akka.grpc.gen.scaladsl.play.PlayScalaClientCodeGenerator,
-      akka.grpc.gen.scaladsl.play.PlayScalaServerCodeGenerator,
+    ReflectiveCodeGen.extraGenerators ++= List(
+      "akka.grpc.gen.scaladsl.ScalaMarshallersCodeGenerator",
```

Comment:
Strings? 😞

Reply:
The reason we use strings for `akka.grpc.gen.scaladsl.play.PlayScalaServerCodeGenerator` here is to get around a circular dependency: `akka.grpc.gen.scaladsl.play.PlayScalaServerCodeGenerator` is part of the `play-grpc-generators` subproject, but we want to use it in the *build* of the `play-grpc-testdata` subproject (rather than as a dependency of the subproject itself).

That is what the `ReflectiveCodeGen` helps with, and why it can't refer to the `akka.grpc.gen.scaladsl.play.PlayScalaServerCodeGenerator` by class: when interpreting the build, the project itself isn't compiled yet so the class isn't available yet.

Classes from akka-grpc such as `akka.grpc.gen.scaladsl.ScalaMarshallersCodeGenerator` could probably be referred to by class, but that would mean making `ReflectiveCodeGen` more complicated to support both classes and strings, which I'm not sure it worth it.

Reply:
Ok, I got it. 

And what about the `AkkaGrpc.Java`?

Reply:
That's possible, updated

Reply:
Done!